.storybook/main.js

```diff
@@ -2,10 +2,11 @@ const path = require('path');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
 module.exports = {
-  stories: [
-    '../src/openforms/js/**/*.stories.mdx',
-    '../src/openforms/js/**/*.stories.@(js|jsx|ts|tsx)',
-  ],
+  core: {
+    disableTelemetry: true,
+  },
+  stories: ['../src/openforms/js/**/*.mdx', '../src/openforms/js/**/*.stories.@(js|jsx|ts|tsx)'],
```

Comment:
it has! `*.stories.mdx` has become `*.mdx` and storybook treats them differently

Then prettier puts it on a single line because now it fits within the max chars on a single line rule :clown_face: